### PR TITLE
[v1.0] Jenkins backport

### PR DESCRIPTION
**Pull request by eloycoto**
Fix on v1.0 branch the artifacts issue.

Signed-off-by: Eloy Coto <eloy.coto@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/4451)

### CODE REVIEW

**Comment by eloycoto** [comment]
test-me-please

**Comment by eloycoto** [comment]
test-me-please

File: ginkgo.Jenkinsfile [hunk 1]
```diff
@@ -85,7 +85,7 @@ pipeline {
                     // rm -rf ${GOPATH}/src/github.com/cilium/cilium
                     sh 'cd test/; ./post_build_agent.sh || true'
                     sh 'cd test/; ./archive_test_results.sh || true'
-                    archiveArtifacts artifacts: "test_results_${JOB_BASE_NAME}_${BUILD_NUMBER}.zip", allowEmptyArchive: true
```
Comment (ianvernon) [inline ginkgo.Jenkinsfile:88]: The change looks fine, but the commit message needs to be more descriptive than saying this addresses an 'issue'. Most commits address issues, so this is somewhat of a vacuous statement. Please be sure to state what issue is actually being addressed within the commit itself, as it makes the commit history easier to follow.

File: ginkgo.Jenkinsfile [hunk 2]
```diff
@@ -85,7 +85,7 @@ pipeline {
...
+                    archiveArtifacts artifacts: "test_results_${JOB_BASE_NAME}_${BUILD_NUMBER}.tar", allowEmptyArchive: true
```
Comment (aanm) [inline ginkgo.Jenkinsfile:88]: Why is this a .tar? shouldn't this be kept a zip file so we can easily upload it to a GH-Issue?

**Review by aanm — CHANGES_REQUESTED** [review]

File: ginkgo.Jenkinsfile (hunk 2 above)
Comment (eloycoto) [reply ginkgo.Jenkinsfile:88]: That feature was introduced after 1.0 was released.

**Comment by eloycoto** [comment]
@ianvernon Fixed.

**Comment by eloycoto** [comment]
test-me-please

**Review by ianvernon — APPROVED** [review]

File: ginkgo.Jenkinsfile (hunk 2 above)
Comment (aanm) [reply ginkgo.Jenkinsfile:88]: Can't we backport that feature to 1.0?

Comment (eloycoto) [reply ginkgo.Jenkinsfile:88]: I don't think that add a real value, the data is in there and the backport will be difficult. IMHO.

**Review by tgraf — APPROVED** [review]

**Pull request merged**